pywinauto/unittests/test_win32controls.py

```diff
@@ -647,6 +673,42 @@ def test_menu_handle(self):
         self.assertNotEquals(0, handle)
 
 
+class StaticTestCases(unittest.TestCase):
+
+    """Unit tests for the StaticWrapper class"""
+
+    def setUp(self):
+
+        """Start the sample application. Open a tab with ownerdraw button."""
+
+        # start the application
+        self.app = Application().Start(os.path.join(mfc_samples_folder, u"RebarTest.exe"))
+        # open the Help dailog
+        self.app.active_().TypeKeys('%h{ENTER}')
+
+    def tearDown(self):
+
+        """Close the application after tests"""
+
+        self.app.kill_()
+
+    def test_NeedsImageProp(self):
+
+        """test a regular static has no the image property"""
+
+        active_window = self.app.active_()
+        self.assertEquals(active_window.Static2._NeedsImageProp, False)
+        self.assertNotIn('Image', active_window.Static2.GetProperties())
+
+    def test_NeedsImageProp_ownerdraw(self):
+
+        """test whether an image needs to be saved with the properties"""
+
+        active_window = self.app.active_()
+        self.assertEquals(active_window.Static._NeedsImageProp, True)
+        self.assertIn('Image', active_window.Static.GetProperties())
+
+

```

Comment:
I do not expect an instance or subclass assignment cause baseclass's attribute change.